tests/e2e/helper_test.go

```diff
@@ -134,9 +133,9 @@ func setupDSCInstance(name string) *dscv1.DataScienceCluster {
 						ManagementState: operatorv1.Removed,
 					},
 				},
-				DataSciencePipelines: datasciencepipelines.DataSciencePipelines{
-					Component: componentsold.Component{
-						ManagementState: operatorv1.Removed,
+				DataSciencePipelines: componentsv1.DSCDataSciencePipelines{
+					ManagementSpec: components.ManagementSpec{
+						ManagementState: operatorv1.Managed,
```

Comment:
@lburgazzoli  I seen in Wen's PR she changed this to Managed for Ray like the Dashboard, just wondering if that is correct?